docker/volumes/mongo-fca-low/scripts/db-states/_default/idp.js

```diff
@@ -224,6 +229,45 @@ const fia = {
     ],
     fqdns: [],
     isRoutingEnabled: true,
+    isEntraID: false,
+  },
+
+  'FIA-ENTRA-LOW': {
```

Comment:
Thought: Is this useful, or should the data in the [infra-legacy](https://github.com/proconnect-gouv/infra-legacy/blob/701435ee8d810428cc003015ac9438e47be5674a/ansible/roles/docker-apps/templates/mongo/mongo-init-db.js.j2#L115-L151) repository be enough?